docs/json-codec-backend/codec-docs.md

```diff
@@ -0,0 +1,67 @@
+## Json Codec Backend
+The purpose of this documentation is to give a explanation of how the Json Codec Backend works and how to use it.
+The Json Codec backend is a tool that generates encoders and decoders from a Morphir IR using the Circe JSON library.
+
+### How to use the JSON Codec Backend
+
+The first step to using the JSON Codec backend is to add the Circe JSON library to your project. This can be done in two ways:
+1. Add the Circe JSON dependency through your build tool eg. Sbt, Mill etc.
+2. Add ```morphir-jvm``` as a dependency to your project.
```

Comment:
I don't think adding Circe as a dependency will be enough since the generated code is referring to codecs in `morphir-jvm`, right? Also, the second bullet-point will need to be more specific down to which exact maven library they should be depending on and in which maven repos it's available.